types/prosemirror-markdown/index.d.ts

```diff
@@ -54,6 +54,13 @@ export interface TokenConfig {
      * When true, ignore content for the matched token.
      */
     ignore?: boolean;
+
+    /**
```

Comment:
Please also update the version number in the line

```typescript
// Type definitions for prosemirror-markdown 1.0
```

at the top of this file to (at least) since this new property was introduced in that version.

Reply:
Added!